src/containers/RockAndPebble/index.js

```diff
@@ -108,8 +108,10 @@ const RockAndPebble = ({ user,
         </div>
           { pendingPebbles && pendingPebbles.length >= 1 && 
           <>
-            <div className='RockAndPebble--div'>        
-            <h2 className='RockAndPebble--header--h2'>Your Pending Pebble(s)</h2>
+            <div className='RockAndPebble--div'>
+              <div class="RockAndPebble--header--div">       
```

Comment:
Haha I swear that I had this header div here and couldn't get it to display the pebbles correctly. Glad it worked for you!

Reply:
I just had to ask nicely